docs/Minimal_Build__Advanced_Usage_with_NNAPI.md

```diff
@@ -1,77 +1,121 @@
-# ONNX Runtime Minimal Build: Advanced Usage with NNAPI
+# ONNX Runtime Mobile Build: Advanced Usage with NNAPI
+
+The ONNX Runtime Mobile with the NNAPI execution provider can be used to execute ORT format models on Android platforms. This document explains the different options available to optimize the ORT model for exection with NNAPI. 
+
+Review the details for using NNAPI with ONNX Runtime Mobile [here](docs/ONNX_Runtime_for_Mobile_Platforms.md#using-nnapi-with-onnx-runtime-mobile).
```

Comment:
Guess link is broken. Shouldn't need the 'docs/' prefix

Reply:
updated.